CRM/Activity/BAO/Activity.php

```diff
@@ -1319,6 +1321,7 @@ public static function sendSMSMessage(
     $recipient = $toPhoneNumber;
     $smsProviderParams['contact_id'] = $toID;
     $smsProviderParams['parent_activity_id'] = $activityID;
+    $smsProviderParams['dao_entity_id'] = $entityID;
```

Comment:
I think drop the `dao` and change to just `entity_id` - seems more like other patterns

Reply:
Thank you for the feedback. I removed it now